templates/index.html

```diff
@@ -21,35 +21,77 @@
-  class="w-100 ma0 h-100" frameborder="0" allow="autoplay;fullscreen" allowfullscreen="" style="
-"></iframe> -->
+    <!-- <iframe src="https://player.vimeo.com/video/365353672?background=1"
+    class="w-100 ma0 h-100" frameborder="0" allow="autoplay;fullscreen" allowfullscreen="" style="
+  "></iframe> -->
 </section>
 
-<nav class="home full c4-10-lg mt6 mb5 grid12">
-  <section class="full half-1-md half-1-lg">
-
+<nav class="home full c3-11-lg mt6 grid12">
+  <section class="full mb6">
+    <div>
+      It's a simple, durable computer you own 100% and forever.
+    </div>
+  </section>
+  <aside class="full half-1-md half-1-lg mb6">
     <ul>
-      <li><a href="/understanding-urbit/">Introduction -></a></li>
+      <li><a class='ul-black' href="/understanding-urbit/">Introduction</a></li>
+      <li class=""><a class="blue2 ul-blue" href="/docs">Docs</a></li>
+      <li><a class="green2 ul-green" href="/install">Install</a></li>
+    </ul>
+  </aside>
+</nav>
 
-      <li class="mt4"><a href="/essays">Essays</a></li>
-      <li><a href="/updates/">Updates</a></li>
-      <li><a href="/media">Media</a></li>
-      <li><a href="/faq/">FAQ</a></li>
+<article class="full-bleed full grid12 mt6">
+  <nav class="full grid12 mt4 mb6">
+    <section class="full c3-7-lg">
+      <h1 class="mb3">Understand Urbit</h1>
+      <p class="lh-copy mt1 mb4">For a quick explanation of what Urbit is and how Urbit works, see the <a href="/understanding-urbit">introduction</a>.
+      If you're more technical, feel free to skip ahead to the <a href="/understanding-urbit/technical-overview">technical overview.</a></p>
+    <ul class="full c4-7-lg">
+        <li class="mt2"><a class="b--black bb" href="/understanding-urbit/">Introduction</a></li>
+        {% set posts = get_section(path="understanding-urbit/_index.md") %}
+        {% for page in posts.pages %}
+        <li class="mt2"><a class="b--black bb" href="{{ page.permalink }}">{{ page.title }}</a></li>
+        {% endfor %}
+    </ul>
+  </section>
 
-      <li class="mt4"><a class="blue2" href="/docs">Docs ↗</a></li>
-      <li><a class="green2" href="/install">Install ↗</a></li>
+  <section class="full c7-11-lg">
+    <h1 class="mb3">Learn More</h1>
+    <p class="lh-copy mt1 mb4">For a quick explanation of what Urbit is and how Urbit works, see the <a href="/understanding-urbit">introduction</a>.
+    If you're more technical, feel free to skip ahead to the <a href="/understanding-urbit/technical-overview">technical overview.</a></p>
+    <ul class="full c4-7-lg">
+        <li class="mt2"><a class="b--black bb" href="/understanding-urbit/">Introduction</a></li>
+        {% set posts = get_section(path="understanding-urbit/_index.md") %}
+        {% for page in posts.pages %}
+        <li class="mt2"><a class="b--black bb" href="{{ page.permalink }}">{{ page.title }}</a></li>
+        {% endfor %}
     </ul>
+  </section>
 
+  <section class="c3-7-lg mt6">
```

Comment:
needs `full` for smaller breakpoints

Reply:
index.html should be fully responsive now